praw/models/reddit/mixins/inboxable.py

```diff
@@ -34,6 +71,18 @@ def mark_unread(self):
         .. note:: This method pertains only to objects which were retrieved via
                   the inbox.
 
+        Example usage:
+
+        .. code:: python
+
+           inbox = reddit.inbox(limit=5)
+
+           # mark last 5 inbox items as unread
```

Comment:
Similar to above, perhaps it makes sense to only mark a single message as unread since `inbox.mark_read` exists for bulk applications.